### ISSUE TITLE
Hosting Pynecone apps

### ISSUE THREAD

**Issue by R-Taneja** [reflex-dev/reflex#1047]
Since Pynecone's deploy feature isn't available yet, I'm reading through the documentation for self-hosting. I'm not super familiar with servers (I'm used to using Vercel for web hosting), so I wanted to ask for some help deploying my finished Pynecone app. I'm looking to be able to access and use it from a custom domain. Ideally, I'd like to not pay anything for a server (very simple site with very few expected users - it consists of a single page with a file upload box and a few API calls). I tried using Firebase Hosting, but it asked for a public/build folder and I wasn't sure what to select. Thanks!

In short: I built an app in Pynecone and want to get it out to the world to display on my portfolio. What's the fastest, cheapest, and easiest way to do that?

Edit: a custom domain isn't a requirement, but a nice-to-have!

**Comment by FHU-yezi** [comment]
Refer to our self-hosting page, just choose a service provider that support Docker image, and use our demo Dockerfile to deploy it.

**Comment by R-Taneja** [comment]
I'm not familiar with Docker or these service providers really. Would you be able to name a few service providers that support Docker images? Where can I find the demo Dockerfile?

**Comment by FHU-yezi** [comment]
> @FHU-yezi I'm not familiar with Docker or these service providers really. Would you be able to name a few service providers that support Docker images? Where can I find the demo Dockerfile?

Demo Dockerfile is at the `docker-example` folder, sorry that I can't list any of providers, since I use my own server to run Docker images.

**Comment by iron3oxide** [comment]
I've had good experiences with [fly](https://fly.io/). They don't have a free tier but a free resource allowance per account which your app is unlikely to exceed unless you get very famous all of a sudden. Their CLI tool, web dashboard and docs are all very helpful and the former do a lot of the heavy lifting for you. You'll still want to have at least a basic understanding of Docker and networking though, no matter which hosting provider you choose; so I recommend reading up on that a little.
